plugins/plugin-hosted/codenvy-machine-hosted/src/main/java/com/codenvy/auth/aws/ecr/AwsEcrAuthResolver.java

```diff
@@ -96,19 +100,20 @@ public AuthConfig getXRegistryAuth(@Nullable String registry) {
     public Map<String, AuthConfig> getXRegistryConfig() {
         Map<String, AuthConfig> dynamicAuthConfigs = new HashMap<>();
 
-        AuthConfig authConfig = getXRegistryAuth(awsInitialAuthConfig.getEcrHostname());
-        if (authConfig != null) {
-            dynamicAuthConfigs.put(awsInitialAuthConfig.getEcrHostname(), authConfig);
+        for (String registry : awsEcrInitialAuthConfig.getAuthConfigs().keySet()) {
+            AuthConfig authConfig = getXRegistryAuth(registry);
+            if (authConfig != null) {
+                dynamicAuthConfigs.put(registry, authConfig);
+            }
         }
 
         return dynamicAuthConfigs;
     }
 
     @VisibleForTesting
-    String getAwsAuthorizationToken() {
+    String getAwsAuthorizationToken(String keyId, String keyValue) {
```

Comment:
I think that Amazon doesn't have such a term **registry key value**. Please use terminology used before: accessKey, accessKeyId or something else if needed.